src/app/shared/pipes/duration.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({
+    name: "duration"
+})
+export class DurationPipe implements PipeTransform {
+
+    transform(min: number): string {
+        const MINUTES_IN_HOUR: number = 60;
+
+        if (min < 60) {
```

Comment:
you have declared constant 2 lines above)

Reply:
fixed :)